src/Chapter1.hs

```diff
@@ -641,7 +656,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n < 0 = firstDigit (0 - n)
```

Comment:
Interesting!
You could use `abs` function on `n` straightaway to have only positive numbers as well 🙂